code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1356,3 +1363,6 @@
 	var/datum/computer_file/program/robotact/program = modularInterface.get_robotact()
 	if(program)
 		program.force_full_update()
+
+/mob/living/silicon/robot/get_eye_protection() 
```

Comment:
is this actually called anywhere?

Reply:
I tested it and it works. It's in flash_act, normally just returns 0 for silicons